app/wallets.html

```diff
@@ -40,6 +41,7 @@ <h2>$( wallets.desktop.title )$</h2>
         <a class="d-block" href="https://electroncash.org" target="_blank">Electron Cash</a>
         <a class="d-block" href="https://pokket.cash" target="_blank">Pokket</a>
         <a class="d-block" href="https://melis.io" target="_blank">Melis Wallet</a>
+        <a class="d-block" href="https://paytaca.com" target="_blank">Paytaca</a>
```

Comment:
Can you add "Chrome only"? I don't think you have dedicated apps for each OS and stuff.